x/expiration/spec/01_concepts.md

```diff
@@ -0,0 +1,101 @@
+# Concepts
+
+The expiration module manages asset expirations.
+
+<!-- TOC -->
+  - [Assets](#assets)
+  - [Expiration Metadata](#expiration-metadata)
+    - [Module Asset ID](#module-asset-id)
+    - [Owner](#owner)
+    - [Expiration Time](#expiration-time)
+    - [Deposit](#deposit)
+    - [Message](#message)
+  - [Creating Expirations on Assets](#creating-expirations-on-assets)
+  - [Extending Expirations on Assets](#extending-expirations-on-assets)
+  - [Owner Enforced Expiration](#owner-enforced-expiration)
+  - [Externally Enforced Expiration](#externally-enforced-expiration)
+  - [Authz](#authz)
+
+
+---
+## Assets
+
+When storing assets on-chain, there is an expense in terms of node memory and processing 
+that is associated with each added asset. Without some method of expiration, the active state 
+of the system would grow until it eventually becomes too large to be efficiently managed and 
+system performance would degrade.
+
+## Expiration Metadata
+
+Expiration metadata is added to the system as part of the asset creation process and 
+is customized on a per-asset-type basis.
+
+### Module Asset ID
+
+the module asset ID is a bech32 address string or a [MetadataAddress](../../metadata/spec/01_concepts.md) 
+that assigned to each asset on chain.
+
+### Owner
+
+The owner is a bech32 address string that determines the rightful owner to a particular expiring asset on chain.
+
+### Expiration Time
+
+The expiration time determines when an asset on chain will expire and is up for removal.
```

Comment:
> up for removal

You can improve the clarity by changing this to "ready for removal".

Reply:
fixed.